docs/install.rst

```diff
@@ -107,66 +105,93 @@ dependencies before doing the install
      $ conda activate beast
      $ pip install .
 
-If you would like to modify beast, you may want to use links instead of
-installing, which is best done by replacing the last line with
+
+.. _Dev_install:
+
+Installation for Developers
+===========================
+
+If you plan on modifying the ``beast`` in addition to running the code, it may
+be useful to create a development installation. First, create a fork of the
+official ``beast`` repository and clone it:
+
+.. code-block:: console
+
+   $ git clone https://github.com/YourName/beast.git
+
+Optionally, you can rename this cloned copy:
+
+.. code-block:: console
+
+   $ git clone https://github.com/YourName/beast.git beast-YourName
+
+Set the value of the fork's 'upstream' to the official distribution so you
+can incorporate changes made by others to your development fork. In the clone
+of your fork, run the following:
 
 .. code-block:: console
 
-     $ pip install -e .
+   $ git remote add upstream https://github.com/BEAST-Fitting/beast.git
+
+In order to run a development installation, navigate to the directory in your
+``beast`` repository that contains `setup.py`, and run:
+
+.. code-block:: console
+
+   $ pip install -e .
+
+Alternatively, you can perform a development install directly through Python
+with:
+
+.. code-block:: console
+
+   $ python setup.py develop
```

Comment:
This no longer works.  It should be `pip install -e .`

Reply:
Ok, I removed it.